kubernetes/running-single-container.md

```diff
@@ -0,0 +1,118 @@
+# Running a single container
+
+[comment]: # (TODO: Environment Setup Guide / Reference?)
+
+Let's say you just want to start one *one* container with Kubernetes. How would you do that?
+
+## Start a deployment
+
+First you need to define a deployment. This can be done a few ways, the fastest is with `kubectl run $DEPLOYMENT_NAME --image=$IMAGE --port=$PORT --replicas=$NUM_REPLICAS`. For example:
```

Comment:
@robszumski I know from experience the structure of the k8s commands can be confusing so I figured adding a template might help.  If it really clutters the document we can get rid of it before merge, but for now I'd like to keep it.
I added everything else in this thread.